src/Form/AdminSettingsForm.php

```
@@ -211,14 +211,19 @@ private function buildIdentityForm() {
     $form['default_identity_type'] = [
       '#type' => 'textfield',
       '#title' => t('Default Identity Type'),
-      '#description' => t('The identity type category to use by default. Leave this field blank to default to <ins>@default</ins>. Example: <ins>@default_identity_type_display_value</ins> is "?@identity_parameter_display_value=jdoe01&@identity_type_parameter_display_value=<ins>@default_identity_type_display_value</ins>", while blank is the same as "?@identity_parameter_display_value=jdoe01&@identity_type_parameter_display_value=<ins>@default</ins>"', [
+      '#description' => t('The Lift Profile Manager identifier type to be used by default. Example using <strong>@default_identity_type_display_value</strong>: a visitor may visit the site through ?@identity_parameter_display_value=jdoe01 and omit the "@identity_type_parameter_display_value" query, and Lift will automatically identify this visitor as "jdoe01" of <strong><ins>@default_identity_type_display_value</ins></strong></strong> type. Leave this field blank to default to <strong>@default</strong> identity type.', [
         '@default' => 'email',
         '@identity_parameter_display_value' => $identity_parameter_display_value,
         '@identity_type_parameter_display_value' => $identity_type_parameter_display_value,
         '@default_identity_type_display_value' => $default_identity_type_display_value,
       ]),
       '#default_value' => $identity_settings['default_identity_type'],
       '#placeholder' => SettingsHelper::DEFAULT_IDENTITY_TYPE_DEFAULT,
+      '#states' => [
+        'visible' => [
+          ':input[name="identity[identity_parameter]"]' => ['!value' => ''],
+        ],
+      ],
```

Adding missing behaviour - "Default identity type parameter" field should hide if there is no value in "Identity parameter".